CMakeLists.txt

```diff
@@ -198,17 +196,15 @@ add_subdirectory(lsp)
 # build libco
 add_subdirectory(third-party/libco)
 
+add_subdirectory(third-party/zstd EXCLUDE_FROM_ALL)
+
 # build SDL
+include(SDLOptions)
 add_subdirectory(third-party/SDL EXCLUDE_FROM_ALL)
-include_directories(third-party/SDL/include)
-
-# build glfw library
-add_subdirectory(third-party/glfw EXCLUDE_FROM_ALL)
-add_subdirectory(third-party/zstd EXCLUDE_FROM_ALL)
 
 # build imgui
 include_directories(third-party/glad/include)
-include_directories(third-party/glfw/include)
+include_directories(third-party/SDL/include)
```

Comment:
after looking a bit, I think we might be better off building it.

Reply:
let's leave it like you have it for now and see how things go

Reply:
for sure, my understanding for why they encourage dynamic linking is so that a newer version of SDL can be used on an old game.  Steam apparently provides their own SDL, so if your game hasn't received an update in a million years, it can still use the latest SDL.  Atleast that's one of the benefits, which isn't really an issue for us.